plugins/discourse-local-dates/config/settings.yml

```
@@ -1,4 +1,7 @@
 plugins:
+  discourse_local_dates_email_format:
+    default: "YYYY/MM/DD HH:mm:ss"
```

I think the default should be the ISO 8601 date format.

```suggestion
    default: "YYYY-MM-DD HH:mm:ss"
```

I wonder if this should be a localizable string instead of a site setting.

---

isn't it automatically localisable due to moment.js?